ding/data/__init__.py

```diff
@@ -0,0 +1,2 @@
+from torch.utils.data import Dataset, DataLoader
```

Comment:
Are there plans to overwrite dataset of torch in the future?

Reply:
yes, it would be the subclass of torch `Dataset` and `DataLoader` or themselves.